assets/app/scripts/controllers/newfromtemplate.js

```diff
@@ -17,6 +17,7 @@ angular.module('openshiftConsole')
       $location.url(redirect);
     }
 
+    // FIXME: This is v1beta1, but it's what's in the ruby-helloworld-sample template
```

Comment:
I think https://github.com/openshift/origin/pull/2486 already updated the templates